### DISCUSSION TITLE
Bulk retry mails in suppression list

### DISCUSSION THREAD

**Discussion by vishnus** [postalserver/postal#1467] [Feature suggestions]
Hi team,

We have few hundred email ids in the suppression list. Mostly because of an invalid dns record we had at our end. Now we'd like to retry sending the mail to all of them. Is there any way we can release messages in bulk? Is there anyway to remove all IDs from suppression list in one go? (even if I can do it through DB that would help a lot for now)

**Comment by mohan-nishit** [comment]
Removal from the DB is easy if you are OK with using the MySQL terminal. If you execute `SHOW DATABASES;` MySQL should show you the list of databases. Postal saves its data for each server in a different DB (like postal-server-1, postal-server-2 etc.). Execute a `USE` statement for your DB, and then `SHOW TABLES;` should show you the tables being used.

To answer your question, the table for suppressions is `suppressions`. So a basis select statement should give you a list of IDs in suppression. If you are simply looking to clear the list, a `TRUNCATE SUPPRESSIONS;` statement should do this. I'd suggest backing up your DB first.
